### ISSUE TITLE
Read files upload in tmp

### ISSUE THREAD

**Issue by SnowRunescape** [Intervention/image#1244]
**Describe the bug**
When uploading the image named "nome_imagem.png," it is saved as "/tmp/phpWdHqFC," and when passing it to $manager->read(), it gives the error shown in the image below.

![image](https://github.com/Intervention/image/assets/52182057/09219aa1-8477-4712-99c5-41b57a79c087)

It seems that the current version obtains the image format from the file name extension, and version 2.x did not have this issue.

**Code Example**
````php
$manager = ImageManager::gd();

$image = $manager->read($file->getRealPath());
$image->scale($width, $height);
$image->save();
````

**Environment (please complete the following information):**
 - PHP Version: 8.2
 - OS: Windows 10
 - Intervention Image Version: 3.2
 - GD or Imagick: GD

**Comment by SnowRunescape** [comment]
Correction, the problem is in the save();

**Comment by olivervogel** [comment]
Thanks for reporting. I have extended the method in https://github.com/Intervention/image/commit/b4597236d46a11c67576ecfa649eba396c0b985cso that the MIME type is used if there is no file extension available.

Will be an a release soon.

**Comment by olivervogel** [comment]
Released in `3.2.1`.
